src/action/mod.rs

```diff
@@ -15,6 +15,7 @@
 //! instance of the corresponding request type and send some (zero or more)
 //! instances of the corresponding response type.
 
+pub mod platform_info;
```

Comment:
This doesn't work, because my file is named `platform_info.rs` and if I change `platform_info` to `platform` I get compilation error. Or, maybe, I don't understand you correctly.

Reply:
I mean, you should rename the module (both the file and this line) to be just `platform`.